test/local/metrics/events.js

```diff
@@ -50,9 +50,9 @@ describe('metrics/events', () => {
       .then(() => {
         assert.equal(log.error.callCount, 1, 'log.error was called once')
         const args = log.error.args[0]
-        assert.equal(args.length, 1, 'log.error was passed one argument')
-        assert.deepEqual(args[0], {
-          op: 'metricsEvents.emit',
+        assert.lengthOf(args, 2, 'log.error was passed two argument')
```

Comment:
I'm wary of suggesting it in case you haven't got the energy, but if we're switching to `lengthOf` we should definitely get rid of the third argument on these I think...